src/string/slice.ts

```diff
@@ -1,5 +1,15 @@
-import { Kind, Type, String, Number, List } from '..'
+import { List, Kind, Type, Number, String } from '..'
 
```

Comment:
I think we might want to make a breaking change so this corresponds to `List.Slice`. What do you think?